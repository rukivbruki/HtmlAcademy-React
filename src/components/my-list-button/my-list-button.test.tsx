import * as React from "react";
import * as renderer from "react-test-renderer";
import MyListButton from "./my-list-button";

const props = {
  isFavorite: true,
  isAuth: true,
  onClick: () => void 0
};

it(`MyListButton renders correctly`, () => {
  const tree = renderer.create(<MyListButton {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
