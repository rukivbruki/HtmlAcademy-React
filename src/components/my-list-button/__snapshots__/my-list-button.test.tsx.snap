// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyListButton renders correctly 1`] = `
<button
  className="btn btn--list movie-card__button"
  onClick={[Function]}
  type="button"
>
  <svg
    height="14"
    onClick={[Function]}
    viewBox="0 0 18 14"
    width="18"
  >
    <use
      xlinkHref="#in-list"
    />
  </svg>
  <span>
    My list
  </span>
</button>
`;
